### PR TITLE
creating a pull request => to check whether a number is prime or not in php

### LINKED ISSUE #249: Check whether a given number is a prime number or not
I am going to work on this.  I will create a pull request

### PR DESCRIPTION
**Pull request by UthpalaPitawela**
This pull request fixes #249

### CODE REVIEW

File: is_prime/is_prime.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<html>
```
Comment (t2013anurag) [inline is_prime/is_prime.php]: use 2 spaces indentation between all the html tags.

File: is_prime/is_prime.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  if($number % $i == 0){
+    $j++;}
+}
+
+if($j==2){
+ print("prime");
+}else{
+ print("not a prime");
+}
+?>
```
Comment (t2013anurag) [inline is_prime/is_prime.php]: add a new line

File: is_prime/is_prime.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+</body>
+</html>
+
+<?php
+$number = $_POST['number'];
+
+$j=0;
+for($i=1;$i<=$number;$i++){
+  if($number % $i == 0){
+    $j++;}
```
Comment (t2013anurag) [inline is_prime/is_prime.php]: bracket in new line,.

**Comment by UthpalaPitawela** [comment]
I have made the changes.

**Pull request merged**